samples/LoggingTracer/LoggingTracer.Demo.AspNetCore/Startup.cs

```diff
@@ -8,12 +8,23 @@ namespace LoggingTracer.Demo.AspNetCore
     using Microsoft.AspNetCore.Hosting;
     using Microsoft.AspNetCore.Http;
     using Microsoft.Extensions.DependencyInjection;
+    using OpenTelemetry.Collector.AspNetCore;
+    using OpenTelemetry.Collector.Dependencies;
 
     public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddLoggingTracer();
+            services.AddOpenTelemetry(() =>
+            {
+                var tracerFactory = new LoggingTracerFactory();
```

Comment:
Usually you would use the builder overload. Because this sample completely re-implements the factory, it and the collectors have to be created manually.